tools/accuracy_checker/accuracy_checker/adapters/retinaface.py

```diff
@@ -238,3 +236,20 @@ def landmark_pred(boxes, landmark_deltas):
             pred[:, i, 1] = landmark_deltas[:, i, 1] * heights + ctr_y
 
         return pred
+
+    @staticmethod
+    def get_scale(meta):
+        if 'scale_x' in meta:
+            return meta['scale_x'], meta['scale_y']
+        original_image_size = meta['image_size'][:2]
+        image_input = [shape for shape in meta['input_shape'].values() if len(shape) == 4]
+        assert image_input, "image input not found"
+        image_input = image_input[0]
```

Comment:
Need to verify ythat there's only one image here, as well.

Reply:
added